### PR TITLE
add in Contributors list : Update chapter01.md

### PR DESCRIPTION
**Pull request by ASawwaf**
add in Contributors list

### CODE REVIEW

File: whitepaper/chapter01.md [hunk 1]
```diff
@@ -143,4 +143,4 @@ As a result, telcos will be able to freely test, design, deploy, provision, revi
...
+Contributors: [Bill Mulligan](https://github.com/xmulligan),[Dan Kohn](https://github.com/dankohn),[Daniel Bernier](https://github.com/dabernie),[Dave Cremins](https://github.com/davecremins),Ervins Kampans,[Frederick Kautz](https://github.com/fkautz),[Gergely Castari](https://github.com/CsatariGergely),Henrik Saveedra Persson,[Herbert Damker](https://github.com/hdamker),[Ildiko Vancsa](https://github.com/ildikov),[Jeff Saelens](https://github.com/jeffsaelens),Khawaja Daniyal,Lei Wang,Saad Sheikh,[Tamas Zsiros](https://github.com/TamasZsiros),[Taylor Carpenter](https://github.com/taylor),[Tom Kivlin](https://github.com/tomkivlin),[W. Watson](https://github.com/wavell), [Ahmed ElSawaf] (https://github.com/ASawwaf)
```
Comment (tomkivlin) [inline whitepaper/chapter01.md]: You need to remove the space between the `]` and the `(`.  Otherwise, no problem with this change.

**Comment by ASawwaf** [comment]
thx

File: whitepaper/chapter01.md [hunk 2]
```diff
@@ -143,4 +143,4 @@ As a result, telcos will be able to freely test, design, deploy, provision, revi
...
+Contributors: [Bill Mulligan](https://github.com/xmulligan),[Dan Kohn](https://github.com/dankohn),[Daniel Bernier](https://github.com/dabernie),[Dave Cremins](https://github.com/davecremins),Ervins Kampans,[Frederick Kautz](https://github.com/fkautz),[Gergely Castari](https://github.com/CsatariGergely),Henrik Saveedra Persson,[Herbert Damker](https://github.com/hdamker),[Ildiko Vancsa](https://github.com/ildikov),[Jeff Saelens](https://github.com/jeffsaelens),Khawaja Daniyal,Lei Wang,Saad Sheikh,[Tamas Zsiros](https://github.com/TamasZsiros),[Taylor Carpenter](https://github.com/taylor),[Tom Kivlin](https://github.com/tomkivlin),[W. Watson](https://github.com/wavell), [Ahmed ElSawaf](https://github.com/ASawwaf)
```
Comment (CsatariGergely) [inline whitepaper/chapter01.md]: s/Castari/Csatari/ thanks

Comment (CsatariGergely) [reply whitepaper/chapter01.md]: Got it. My name my change. I will create a pr to fix it.

Comment (taylor) [reply whitepaper/chapter01.md]: @CsatariGergely fixed in https://github.com/cncf/telecom-user-group/commit/1f218d6a1062b2c4c339e491c8118f7591161388

**Pull request merged**